controllers/Alerts.js

```diff
@@ -0,0 +1,99 @@
+const alertRouter = require('express').Router();
+const mongodb = require('mongodb');
+const Alert = require('../models/Alert');
+
+// get a list of alerts
+alertRouter.get('/list', function(req, res) {
+	Alert.find(function(err, alerts) {
+		if (err) {
+			res.json(err);
+		}
+		const mappedAlerts = alerts.map(q => q.toJSONFor());
+		res.json(mappedAlerts);
+	});
+})
+
+// get a specific alert
+alertRouter.get('/id/:id', function(req, res) {
+  const id = req.params.id;
+  Alert.findById(id, function(err, alert) {
+    if (err) {
+      return res.status(500).json(err);
```

Comment:
Follow one-line statement from other files

Reply:
Is there a reason why there are some conditionals that aren't one-line statements in other files?

Reply:
If the block statement related to a conditional has only one line, we want to have it without the braces